### ISSUE TITLE
Big problem to synchronize our email accounts

### ISSUE THREAD

**Issue by franco27** [nylas-mail-lives/nylas-mail#155]
Hello
In the past at the beginning of 2017, I installed Nylas N1. No slowness for synchronizing email accounts. I regretted having installed the new version: Nylas Mail. A disaster or a horror for synchronization. Because of this problem, I uninstalled and canceled the subscription.

Nylas Mail lives has kept this big inconvenience. too bad ! The synchronization is always also slow.

This morning, I downloaded and installed the version NylasMailSetup-2.2.1-x64.exe
(on Windows 10 x64bits)
My hardware configuration is pretty performance:
- Video card NVIDIA GeForce GT 640  (2 Go) ; System memory 8153 Mo  (DDR3 SDRAM) ; 
Processor type	QuadCore Intel Core i5-3570, 3800 MHz (38 x 100)

After adding 4 e-mail accounts and waiting 8 hours, the synchronization was still not finished!
I specify that my computer was not used during this time or this duration.
Big disappointment again!
Sorry, I uninstalled your software.

Previously, on my second computer, it's an old laptop, with a much less powerful hardware configuration. (Windows10 x32 + Intel Core 2 duo 2.20GHz + RAM 3GB)
I installed Mailspring and the synchronization of these same email accounts were completed after 2 or 3 hours of waiting.

So I hope you will improve your synchronization engine, you have work and good luck!

**Comment by dweremeichik** [comment]
Yeah... This is a known limitation. Unless someone from the community contributes, there is no plan to change this. As you said, Mailspring is better with performance.

**Comment by franco27** [comment]
Thank you for your reply. Unfortunately, I do not know how to do computer programming.

**Comment by jeroen7s** [comment]
can't you borrow the syncing code from mailspring?

**Comment by seesemichaelj** [comment]
Mailsprings syncing code is completely closed source, which is most people's reason to stick with Nylas
